cluster/gce/util.sh

```diff
@@ -2684,10 +2684,12 @@ function kube-up() {
       done
       restart_tp_scheduler_and_controller
 
-      echo "DBG: defining CRD networks.arktos.futurewei.com at all TPs"
-      for num in $(seq ${SCALEOUT_TP_COUNT:-1}); do
-        "${KUBE_ROOT}/cluster/kubectl.sh" --kubeconfig="cluster/kubeconfig.tp-${num}" apply -f "${KUBE_ROOT}/pkg/controller/artifacts/crd-network.yaml"
-      done
+      if [[ "${PROVISION_MODE:-}" == "kube-up" ]]; then
```

Comment:
suggest to use this exiting var:  if [[ "${KUBEMARK_PREFIX:-}" == "" ]]; then

we have this set when kubemark:
KUBEMARK_PREFIX="${KUBEMARK_PREFIX:-.kubemark}"

Reply:
good hint. will take its advantage.